app/views/games/show.html.erb

```diff
@@ -14,9 +14,23 @@
   </div>
 </div>
 
+<script src="https://www.gstatic.com/firebasejs/3.7.6/firebase-app.js"></script>
+<script src="https://www.gstatic.com/firebasejs/3.7.6/firebase-database.js"></script>
+<script>
+  // Initialize Firebase
+  var config = {
+    apiKey: '<%= ENV["apiKey"] %>',
```

Comment:
^ Nevermind, it's not a security issue as long as the database has the proper 'Rules' implemented.